src/components/structures/TagPanel.js

```diff
@@ -60,7 +61,9 @@ const TagPanel = React.createClass({
 
     componentWillUnmount() {
         this.unmounted = true;
+        if (!this.context.matrixClient) return;
```

Comment:
I think we established this was unnecessary?

Reply:
yep